mica/archive/tests/test_aca_hdr3.py

```diff
@@ -63,3 +63,25 @@ def test_MSIDset():
             10679,
         ]
     )
+
+
+def test_two_byte_sum():
+    bytes0 = np.ma.array([0x00, 0xF0, 0x0F, 0xFF, 0xFF], dtype=np.uint8)
+    bytes1 = np.ma.array([0x00, 0x0F, 0xF0, 0xFF, 0xFF], dtype=np.uint8)
+    bytes0[-1] = np.ma.masked
+    bytes1[-1] = np.ma.masked
+
+    # Original code prior to PR #315
+    out1 = (
+        (bytes0.astype("int") >> 7) * (-1 * 65535)
+        + (bytes0.astype("int") << 8)
+        + (bytes1.astype("int"))
+    )
+    assert np.all(out1 == np.ma.array([0, -4080, 4080, 0, 0], mask=[0, 0, 0, 0, 1]))
+
+    # New code in PR #315
+    bytes8_2xN = np.ma.vstack([bytes0, bytes1], dtype=np.uint8)
+    bytes8 = bytes8_2xN.transpose().flatten().copy()
+    ints16 = np.ma.array(bytes8.data.view(">i2"), mask=bytes8.mask[::2])
```

Comment:
I like that this test has the previous buggy code captured for reference, but it seems like the new test should actually call the code in aca_hdr3.  Maybe 
```
    # Current method
    from astropy.table import Table
    slot_data = Table([bytes0, bytes1], names=["byte0", "byte1"])
    ints16 = aca_hdr3.two_byte_sum(["byte0", "byte1"])(slot_data)
```

Reply:
Done.